File: berkeleytime/templates/berkeley_time_template.py
def solve(A: int) -> str:
    """
    Return the berkeleytime value of A.
    
    A: a non-negative integer
    """
    # YOUR CODE HERE
    return 0


def main():
    T = int(input())
    for _ in range(T):
        temp = input().split()
        A = int(temp[0])
        print(solve(A))

if __name__ == '__main__':
    main()

File: berkeleytime/templates/berkeleytime_template.java
import java.io.*;

class Solution {
    /**
     * Return the appropriate text given the contest will start N minutes late.
     * 
     * N: the number of minutes late the contest will start
     */
    static String solve(int N) {
        // YOUR CODE HERE
        return "";
    }
    
    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            int N = Integer.parseInt(in.readLine());
            out.println(solve(N));
        }
        out.flush();
    }
}
